diff --git a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
index 52450f0..446aec8 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
@@ -31,11 +31,13 @@ target_include_directories(
 # https://github.com/react-native-community/discussions-and-proposals/discussions/816
 # This if-then-else can be removed once this library does not support version below 0.76
 if (REACTNATIVE_MERGED_SO)
+  find_library(CPP_SHARED_LIB c++_shared)
   target_link_libraries(
           ${LIB_TARGET_NAME}
           fbjni
           jsi
           reactnative
+          ${CPP_SHARED_LIB}
   )
 else()
   target_link_libraries(
